src/app/_not-found/page.tsx

```diff
@@ -0,0 +1,11 @@
+// app/_not-found/page.tsx
```

Comment:
Nice!  Wonder if we can leverage locale and the associated language translation for this page as well.